systemd/manager.go

```diff
@@ -106,6 +106,13 @@ func (m *systemdUnitManager) Load(name string, u unit.UnitFile) error {
 	if err != nil {
 		return err
 	}
+	if u.HasReverseDependencies() {
```

Comment:
I see. That being so, how about just making it conditional on the presence of this section? It seems a much clearer and less error-prone distinciton...